demo/src/stories/index.js

```diff
@@ -3,15 +3,31 @@ import { storiesOf, action, linkTo } from '@kadira/storybook';
 import Button from './Button';
 import Welcome from './Welcome';
 
-storiesOf('Welcome', module)
+storiesOf('Welcome')
```

Comment:
Removing `module` we will have an isssue with HMR. Shouldn't we try to handle hot reloadings instead?